templates/definition/meter/enphase.yaml

```diff
@@ -66,3 +68,31 @@ render: |
     jq: .production[] | select(.measurementType == "production").whLifetime
     scale: 0.001
   {{- end }}
+  {{- if eq .usage "battery" }}
+  power:
+    source: http
+    uri: http://{{ .host }}/production.json?details=1
+    method: GET
+    {{- if .token }}
+    auth:
+      type: bearer
+      password: {{ .token }}
+    insecure: true
+    {{- end }}
+    jq: .storage[] | select(.type == "acb").wNow
+    scale: 1
```

Comment:
```suggestion
```

Reply:
Removed `scale: 1` for `battery` and also for `grid` and `pv`